rdr_client/work_queue.py

```diff
@@ -3,16 +3,25 @@
 
 import logging
 import pprint
+
 from main_util import configure_logging
 from client import Client
 
 
 def main():
   client = Client()
+  AWARDEE = 'PITT'
+  response = client.request_json('ParticipantSummary?_sync=true&_count=1&_sort=lastModified&awardee'
+                                 '={}'.format(AWARDEE), 'GET')
 
-  response = client.request_json('ParticipantSummary?_sync=true&_sort=lastModified&awardee=PITT',
-                                 'GET')
+  sync_url = response['link'][0]['url']
+  index = sync_url.find('ParticipantSummary')
+  sync_results = client.request_json(sync_url[index:], 'GET')
   logging.info(pprint.pformat(response))
+  print
+  print '--------------      Getting next batch from sync...        -----------------'
```

Comment:
1) I'd have a loop where we sync new results every SYNC_PERIOD seconds, and print the new results each time, either for some fixed number of times or indefinitely until the program is killed. For testing purposes you can make this a few seconds, but the version we check in and share with HPOs should set it to 300 seconds.

2) These print statements happen after the sync actually occurs. You should probably put them beforehand?

Reply:
done